app/javascript/components/modals/WelcomeModal.tsx

```diff
@@ -30,7 +30,10 @@ export default function WelcomeModal({
     },
     {
       onSuccess: () => {
-        handleClose()
+        setOpen(false)
+      },
+      onError: (e) => {
+        console.log('this is an error:', e)
```

Comment:
Was this temporary? I mean: it doesn't offer any value to the student :)

Reply:
Yes, removing it. Thanks!